plugins/inputs/elasticsearch_query/aggregation_parser.go

```diff
@@ -0,0 +1,154 @@
+package elasticsearch_query
+
+import (
+	"fmt"
+
+	"github.com/influxdata/telegraf"
+	elastic "gopkg.in/olivere/elastic.v5"
+)
+
+type resultMetric struct {
+	name   string
+	fields map[string]interface{}
+	tags   map[string]string
+}
+
+func (e *ElasticsearchQuery) parseSimpleResult(measurement string, searchResult *elastic.SearchResult, acc telegraf.Accumulator) error {
```

Comment:
As this function cannot error please remove the return parameter to make this obvious to everybody.

Reply:
And can you please make `acc` the first parameter. It's not really a convention but nice to indicate "Hey we pass the acc down here.".